.github/workflows/publish-android.yml

```diff
@@ -24,9 +24,11 @@ on:
     inputs:
       github_publish:
         description: 'Publish to github packages'
+        type: boolean
```

Comment:
When specifying arguments to a workflow through workflow dispatch it processes any input as a string (its a github limitation for the time being), so we need to compare strings.